cli/azd/pkg/tools/azcli/azcli.go

```diff
@@ -26,6 +26,7 @@ import (
 
 var (
 	ErrAzCliNotLoggedIn          = errors.New("cli is not logged in. Try running \"azd login\" to fix")
+	ErrAzCliNotProperLoggedIn    = errors.New("az cli is not properly logged in. See more here: https://github.com/Azure/azure-dev/issues/197")
```

Comment:
I feel like saying the "cli is not properly logged in" gives the wrong impression as to how to fix this.  The real issue is that this operation will never be allowed due to conditional access rules, so you need to either change something about your environment, the rules themselves or adopt a different principal that doesn't have these restrictions.  I wonder if it makes sense to call out something in the error about "conditional access".  

Maybe something like: `"this operation is not allowed due to conditional access rules for the current user, for more information visit https://github.com/Azure/azure-dev/issues/197"`

We could also consider making that an aka.ms link like `https://aka.ms/azure-dev/conditional-access` which we could point at GH for now but longer term maybe link to a page on docs.microsoft.com that could give better guidance.

Reply:
Agree.  
I was expecting to receive feedback for the error message from the PM office :)  @savannahostrowski  @puicchan 

The only thing this error is looking for is to improve the ux and some guide/next steps about what to do next

Reply:
I agree with Matt. When I saw "cli is not properly logged in", I thought doing `az login` again will resolve my issue. I like "this operation is not allowed due to conditional access rules for the current user, for more information visit https://github.com/Azure/azure-dev/issues/197"

Reply:
+1 to PC and Matt's comment here!